WordPress/gutenberg
# InputControl: Fix incorrect size prop passing to `Text`

## What?

Removes the erroneous passing down of a `size` prop, which was resulting in invalid CSS like `font-size: default`. 

There are no visual changes, because that invalid `font-size` value was being overridden anyway in the CSS cascade.

## Why?

Clean up.

## How?

This bug has been there since the inception of InputControl (#22329). This PR is a simple fix — bug error prevention measures will be discussed separately in #42792.

## Testing Instructions

1. `npm run storybook:dev`
2. Inspect the label element of an InputControl-based component and see that invalid CSS like `font-size: default` aren't being emitted.
